wordpress-mobile/gutenberg-mobile
# Enable reusable block (preview mode)

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/31154

To test:
Check testing instructions from [`gutenberg` PR](https://github.com/WordPress/gutenberg/pull/31154).

PR submission checklist:
